lib/gooddata/rest/connection.rb

```diff
@@ -382,6 +383,47 @@ def get(uri, options = {}, &user_block)
         request(:get, uri, nil, options, &user_block)
       end
 
+      def stream(uri, filename)
+        uri = URI.join(server_url, uri.to_s)
+        puts "Streaming URL #{uri.to_s}"
+
+        q = { headers: @webdav_headers.merge(:x_gdc_authtt => headers[:x_gdc_authtt]) }
+        File.open(filename, 'wb+') do |fout|
+          Net::HTTP.start(uri.host, uri.port,
```

Comment:
Can you describe why cannot we use `RestClient` for this? Is it a bug in `RestClient` or the API?